kolibri/core/tasks/storage.py

```diff
@@ -143,39 +147,71 @@ def mark_job_as_canceling(self, job_id):
         """
         self._update_job(job_id, State.CANCELING)
 
-    def get_next_queued_job(self, queues):
+    def get_next_queued_job(self, queues=None, priority_order=Priority.PriorityOrder):
```

Comment:
Is there a use case for filtering by `queues` for the next queued job now? My assumption was that we would only be using the `queues` field for filtering jobs to return via the API?

Reply:
Yes, it should simplify the code!